core/jazz-ui/src/app/pages/environment-codequality/env-codequality-section.component.ts

```diff
@@ -175,7 +175,7 @@ export class EnvCodequalitySectionComponent implements OnInit {
   }
 
   sonarLink() {
-    window.open(this.selectedMetric.metric.link, '_blank');
+    window.open(this.selectedMetric.link, '_blank');
```

Comment:
This is reverting @rjsughosh's  changes. Can you and @rjsughosh figure out what changes are required and update the PR?